src/main/java/org/commonwl/view/cwl/CWLProcess.java

```diff
@@ -35,9 +35,8 @@ public String toString() {
             case COMMANDLINETOOL:
                 return "CommandLineTool";
             case EXPRESSIONTOOL:
-                return "ExpressionTool";
             default:
-                return super.toString();
+                return "ExpressionTool";
```

Comment:
I wouldn't merge `EXPRESSIONTOOL` with `default` just to improve test courage - change enum to have a `String name`  parameter in the constructor and return its field.